core/modules/filter/js/filter.admin.js

```diff
@@ -316,8 +316,8 @@ Backdrop.FilterStatus.prototype.addHTMLRule = function (rule) {
  * Backdrop.FilterStatus.
  *
  * A text filter rule object describes
- *  1. allowed or forbidden tags: (optional) whitelist or blacklist HTML tags
- *  2. restricted tag properties: (optional) whitelist or blacklist attributes,
+ *  1. allowed or forbidden tags: (optional) allowlist or denylist HTML tags
+ *  2. restricted tag properties: (optional) allowlist or denylist attributes,
  *     styles and classes on a set of HTML tags.
```

Comment:
The replaced words here seem like verbs to me. If so, then this should be:

```suggestion
 *  1. allowed or forbidden tags: (optional) allow or deny HTML tags.
 *  2. restricted tag properties: (optional) allow or deny attributes, styles
 *     and classes on a set of HTML tags.
```

Reply:
...alternatively, we could use `accept`/`reject` here:

```suggestion
 *  1. allowed or forbidden tags: (optional) accept or reject HTML tags.
 *  2. restricted tag properties: (optional) accept or reject attributes, styles
 *     and classes on a set of HTML tags.
```